package.json

```diff
@@ -96,7 +96,7 @@
     "@babel/preset-react": "^7.23.3",
     "@rollup/plugin-babel": "^6.0.4",
     "@rollup/plugin-commonjs": "^25.0.7",
-    "@rollup/plugin-inject": "^5.0.5",
```

Comment:
This was left over from some attempts to fix a prior issue. Removing since it's not actually used.